### PR TITLE
20.2.1 clear release notes

### PR DESCRIPTION
**Pull request by jochman**
## ready

### CODE REVIEW

File: Integrations/integration-CrowdStrikeHost_CHANGELOG.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 ## [Unreleased]
+
+
```
Comment (Itay4) [inline Integrations/integration-CrowdStrikeHost_CHANGELOG.md]: how is this related?

File: Misc/reputations_CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+
```
Comment (Itay4) [inline Misc/reputations_CHANGELOG.md]: how is this related?

File: Scripts/SaneDocReport/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ## [Unreleased]
 -
```
Comment (Itay4) [inline Scripts/SaneDocReport/CHANGELOG.md]: how is this related?

Comment (jochman) [reply Scripts/SaneDocReport/CHANGELOG.md]: added the blank line to not mess with the next release
